script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -828,42 +962,51 @@ Here, I'll give you one extra dose, on the house!
 [sha:836753594415c7d08e1f1e2e560a354f80ea720f]{
 -- Page 337, Offset 36261. Glued. Mods: @x.
 -- あ、でもこれはシャレだから、追加で射つのは止めときなさいねぇ？　今ですら破裂寸前なんだもの。もう一つ上を目指すなんて、ほんと、一瞬で融けちゃうから！”
- Ah, but this one's quite the hit, so I'd have to caution against taking it, okay? You already look like you're about to explode, so if you try to go up another stage, you're seriously going to melt away in an instant!"
+// One last mint wafer
+ Ah, this one packs a real punch though,
+ so I wouldn't recommend shooting up any more than you already have, okay?
+ As things stand you're %{i}thiiis%{/i} close to your breaking point.
+ If you start gunning for the next Rank up, it really will melt your face off!"
```

Comment:
I don't even know. I went through my search history and nothing I looked at makes it clear. I probably just gave up and assumed it was slang because it doesn't make sense. Maybe it's supposed to be like 'wild card'?